gax/src/main/java/com/google/api/gax/rpc/ApiCallContext.java

```diff
@@ -153,6 +156,54 @@
   @BetaApi("The surface for tracing is not stable yet and may change in the future")
   ApiCallContext withTracer(@Nonnull ApiTracer tracer);
 
+  /**
+   * Returns a new ApiCallContext with the given {@link RetrySettings} set.
+   *
+   * <p>This sets the {@link RetrySettings} to use for the RPC. These settings will work in
+   * combination with either the default retryable codes for the RPC, or the retryable codes
+   * supplied through {@link #withRetryableCodes(Set)}. Calling {@link
+   * #withRetrySettings(RetrySettings)} on an RPC that does not include {@link
+   * Code#DEADLINE_EXCEEDED} as one of its retryable codes (or without calling {@link
+   * #withRetryableCodes(Set)} with a set that includes at least {@link Code#DEADLINE_EXCEEDED})
+   * will effectively only set a simple timeout that is equal to {@link
```

Comment:
Nit: When you say "simple timeout" do you mean "constant timeout for each attempt"? This currently sounds pretty vague / open to interpretation of what we think "simple" means. "constant, per-PRC timeout" is still short, but much more clear imo.

Reply:
I've clarified the documentation regarding the 'simple' timeouts and removed the reference to `withTimeout(..)` to prevent any confustion.

Reply:
Thank you! Looks great.